lib/util.ts

```diff
@@ -1,3 +1,20 @@
+import { Readable } from "stream";
+import * as fileType from "file-type";
+const { default: fileTypeStream } = require("file-type-stream");
```

Comment:
Destructuring doesn't work in Node.js 4. Please use a plain `require("file-type-stream").default`.